docs/examples/approval-form.swadl.yaml

```diff
@@ -0,0 +1,56 @@
+id: approval-form
+activities:
+  - send-message:
+      id: sendForm
+      on:
+        one-of:
+          - message-received:
+              content: /send
+          - activity-completed:
+              activity-id: setVariables
+              if: ${variables.loop == true}
+      content: |
+        <messageML>
+          <p>Do you approve the action?</p>
+          <form id="sendForm">
+            <button name="approve" type="action">Approve</button>
+            <button name="reject" type="action">Reject</button>
+          </form>
+        </messageML>
+
+
+  # Form requiring only one reply from the admin
+  # 10s to reply before it expires
+  - execute-script:
+      id: setVariables
+      on:
+        timeout: PT10S
+        form-replied:
+          form-id: sendForm
+          unique: true
+      script: |
+        variables.loop = (sendForm.action == "reject" )
+
+  - send-message:
+      id: sendFeedBackOnReply
+      content: The form has been approved!
+
+  - send-message:
+      id: sendFeedBackOnExpiration
+      content: The form timed out, the workflow will finish immediately
+      on:
```

Comment:
I personally prefer to have the on/event right after the id (and also to be consistent with the activity above)